linter/rules/yesno_field_label_contains_yesno.py

```diff
@@ -0,0 +1,13 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+from re import search, IGNORECASE
+
+
+class YesnoFieldLabelContainsYesno(Rule):
```

Comment:
The name of this class is confusing, I was assuming it meant it _should_ contain yes/no.  Maybe `YesnoFieldLabelShouldNotContainYesno` would be better.